courseaffils/tests/test_settings.py

```diff
@@ -37,7 +37,6 @@
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'courseaffils',
-    'django_markwhat',
```

Comment:
We can remove `django-markwhat` from `test_reqs.txt` as well in this PR, if you're removing it here.

Reply:
Done.